doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -26,8 +53,128 @@ This document covers the realisation aspects of Verification and Certification o
+
+<p align="center"><img src="../figures/RC_certifying_methodlogy_25Nov2019.jpg" alt="Certification Methodology" title="Certification Methodology" width="100%"/></p>
+<p align="center"><b>Figure:</b> Figure: Certification Methodology</p>
+
+<a name="1.6"></a>
+### Assumptions & Dependencies
+
+**Assumptions** NFVI+VNF testing will be considered **Testable** if the follow qualifiers are present in a test execution, and subsequent result:
+
+* Ability to perform Conformance, or Verification of Artifacts to ensure designs (RM/RA/RI) are delivered per specification
+
+* Ability to Control (or manipulate), manifestations of RM/RA/RI for the purposes to adjust the test environment, and respective cases, scenarios, and apparatus, to support actual test validations
+
+* Ability to monitor, measure, and report, Validations performed against a target, controlled system under test
+
+In addition, respective Entrance criteria is a prerequisite which needs to be satisfied for NFVI+VNF to be considered **Testable**. 
+
+**Dependencies** NFVI+VNF verification will rely upon test harnesses, test tools, and test suites provided by upstream OPNFV projects, including dovetaill, yardstick, and Bottleneck. These upstream projects will be reviewed semi-annually to verify they are still healthy and active projects. Over time, the projects representing the certification process may change, but test parity is required if new test suites are added in place of older, stale projects.
+
+* NFVI+VNF verifications will be performed against well defined instance types consisting of a HW and SW Profile, Configured Options, and Applied Extensions (See image.)
+
+<p align="center"><img src="../figures/RC_NFVI_VNF_Instance_Type_25Nov2019.jpg" alt="Instance Type" title="Instance Type" width="100%"/></p>
+<p align="center"><b>Figure:</b> Figure: Instance Type</p>
+
+**Figure:** NFVI+VNF Instance Type
+* Standard compute flavours to be tested are defined in [chapter 4.2.1](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#4.2.1)
+* Performance profiles come in the form of Basic, Network Intensive, and Compute intensive. Refer to [chapter 2.3](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter02.md#2.3) for details on these profiles.
+
+<a name="1.7"></a>
+### Results Collation & Presentation
+
+Test suites will be categorized as functional or performance based. Results reporting will be communicated as a boolean (pass/fail). The pass/fail determination for performance-based test cases will be made by comparing results against a baseline. Example performance-based metrics include, but are not limited to: resource utilization, response times, latency, and sustained throughput per second (TPS).
+
+**Placeholder to document where results will be posted (e.g. Dovetail dashboards.)**
+
+<a name="1.8"></a>
+### Measurements, Monitoring
+
+<a name="1.8.1"></a>
+#### 1.8.1 Performance & Resiliency - Measurements, Testing.
```

Comment:
@mf4716 this is covered in Ch03? https://github.com/cntt-n/CNTT/blob/master/doc/ref_cert/lfn/chapters/chapter03.md#3.3.1 

They can live on either place. would it be possible to check Ch03 and make sure we don't have duplication?

Reply:
Removed RC1 Ch01 1.8 as similar/dup to RC1 Ch3.   Thanks @rabiabdel .